apps/microtvm/ethosu/README.md

```diff
@@ -16,26 +16,24 @@
 <!--- under the License. -->
 
 
-Running TVM on bare metal Arm(R) Cortex(R)-M55 CPU, Ethos(TM)-U55 NPU and CMSIS-NN
```

Comment:
CMSIS-NN is removed from the description, but it remains in run_demo.sh as target.

Reply:
Hi @Aleksei-grovety I'd like to keep CMSIS-NN target for the case if someone want to use that tvmc compilation command for their model. I've added additional comment about possible CMSIS-NN dependency inside `run_demo.sh`